src/txdb.h

```diff
@@ -38,7 +38,7 @@ static const int64_t nDefaultDbCache = 450;
 //! -dbbatchsize default (bytes)
 static const int64_t nDefaultDbBatchSize = 16 << 20;
 //! max. -dbcache (MiB)
-static const int64_t nMaxDbCache = sizeof(void*) > 4 ? 16384 : 1024;
+static const int64_t nMaxDbCache = sizeof(void*) > 4 ? 32768 : 1024;
```

Comment:
Might as well make it `constexpr` while you're touching it

Reply:
Done